packages/@icon-magic/build/src/index.ts

```diff
@@ -35,8 +35,10 @@ const LOGGER: Logger = logger('icon-magic:build:index');
  */
 export async function build(
   iconConfig: IconConfigHash,
-  hashing = true
+  hashing = true,
+  debug = false
 ): Promise<IconSet> {
+  LOGGER.setDebug(debug);
```

Comment:
Pulling Ramitha's comment from other draft PR: https://github.com/linkedinlabs/icon-magic/pull/233#discussion_r624337099

"Can we set this within the cli itself, instead of within each function?"

Reply:
Sounds good, let me try this refactor, Ty

Reply:
Ok, changed in latest commit. Ty both..